lib/inc/drogon/HttpClient.h

```diff
@@ -49,7 +49,7 @@ struct HttpRespAwaiter : public CallbackAwaiter<HttpResponsePtr>
   private:
     HttpClient *client_;
     HttpRequestPtr req_;
-    double timeout_;
+    long double timeout_;
```

Comment:
Why `long double`, is there a use case for it? AFAIK most CPUs don't support it natively so is slower.

Reply:
The runAfter method of the EventLoop class uses long double type, should we change them all to double?
```c++   
 TimerId runAfter(const std::chrono::duration<long double> &delay,
                     const Func &cb)
    {
        return runAfter(delay.count(), cb);
    }
```

Reply:
I think we should change them into `double`. 

I checked. GCC and Clang on x64 treats long double as quad precision.  MSVC treads it as regular double.

Reply:
okay